src/components/tl/mlx5/alltoall/alltoall_coll.c

```diff
@@ -786,12 +835,15 @@ UCC_TL_MLX5_PROFILE_FUNC(ucc_status_t, ucc_tl_mlx5_alltoall_init,
                                                         == a2a->node.asr_rank);
     int                     n_tasks   = is_asr ? 5 : 3;
     int                     curr_task = 0;
+    int                       ppn       = tl_team->a2a->node.sbgp->group_size;
```

Comment:
Maybe align assignments and variables?

Reply:
clang format did that. It is aligned with the line below. Is it ok or do you suggest something else?